landingai/predict.py

```diff
@@ -706,3 +702,13 @@ def _do_inference(
     response.raise_for_status()
     json_dict = response.json()
     return extractor_class.extract_prediction(json_dict)
+
+
+def _add_defualt_query_params(payload: Dict[str, Any]) -> None:
+    """Add default query params to the payload for tracking and analystics purppose."""
+    env_info = get_runtime_environment_info()
+    payload["runtime"] = env_info["runtime"]
+    if is_running_in_pytest():
+        # Don't add extra query params if pytest is running, otherwise it will fail some unit tests
+        return
+    payload["lib_version"] = env_info["lib_version"]
```

Comment:
Why not using `device_version` as LE so we can all use the same dashboards??

Reply:
chatted offline, will continue use `lib_version`